portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -115,41 +110,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Redirect back to the HTML page.
     response.sendRedirect("/comments.html");
   }
-
-  /** Returns a URL that points to the uploaded file, or null if the user didn't upload a file. */
-  private String getUploadedFileUrl(HttpServletRequest request, String formInputElementName) {
+  
+  /**
+   * Returns the String representation of the blobKey of the file uploaded by the user to Blobstore. 
+   * Returns null if the user did not upload a file.
```

Comment:
You might need to fix the wrapping, but this is how we write the `null` return cases.

```suggestion
   * Returns the String representation of the blobKey of the file uploaded by the user to Blobstore or {@code null} if 
   * the user did not upload a file.
```

Reply:
got it